ruby-monday-blog/spec/models/post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it "makes a new post with title" do
```

Comment:
A tool you might also want to check out is [Shoulda Matchers ](https://github.com/thoughtbot/shoulda-matchers). Instead of doing a full multi-line spec like this, it allows you to do one liners like `it { should have_attribute(:title) }` which is cleaner and easier to read for things like attributes. That way, you can focus the pages on most important specs like methods you wrote yourself, as opposed to testing methods given to you by Active Record.